landoui/revisions.py

```diff
@@ -92,17 +96,28 @@ def uplift():
     # Get the list of available uplift repos and populate the form with it.
     uplift_request_form.repository.choices = get_uplift_repos(api)
 
+    # If we return an error and we don't hit a block with a specific problem, consider
+    # the error a server-side issue.
+    return_code = 500
```

Comment:
nit: slightly more readable to set this to `None` here, and then return `return_code or 500` below.